src/DeepgramClient.ts

```diff
@@ -17,6 +18,15 @@ import {
  * @see https://github.com/deepgram/deepgram-js-sdk
  */
 export default class DeepgramClient extends AbstractClient {
+  /**
+   * Returns a new instance of the AuthRestClient, which provides access to the Deepgram API's short-lived token endpoints.
```

Comment:
```suggestion
   * Returns a new instance of the AuthRestClient, which provides access to the Deepgram API's temporary token endpoint.
```

Reply:
just using the terminology we use in Docs here:
 
https://developers.deepgram.com/reference/token-based-auth-api/grant-token